ggcnn_service:
    camera:
        info_topic: /kinect/rgb/camera_info
        depth_topic: /kinect/depth/image_raw
        camera_frame: camera_link
        robot_base_frame: panda_link0
        crop_size: 300
        crop_y_offset: 40
        fov: 65.5
